src/datta_lab_to_nwb/markowitz_gillis_nature_2023/markowitz_gillis_nature_2023behaviorinterface.py

```diff
@@ -1,22 +1,108 @@
 """Primary class for converting experiment-specific behavior."""
-from pynwb.file import NWBFile
-
+import numpy as np
+import pandas as pd
+from pynwb import NWBFile, TimeSeries
+from pynwb.behavior import (
+    BehavioralTimeSeries,
+    CompassDirection,
+    Position,
+    SpatialSeries,
+)
 from neuroconv.basedatainterface import BaseDataInterface
 
 class MarkowitzGillisNature2023BehaviorInterface(BaseDataInterface):
```

Comment:
Would recommend specifying the name of this interface now to 'MarkowitzGillisNature2023PositionInterface` to make this more specific to the data types you're adding below

(don't forget to change file name was you're at it)

Reply:
As it stands rn, this interface adds both position (x, y, height), orientation, and behavioral syllable.  So I think 'behavior' captures all of those pretty well (just 'position' would be incomplete).  Now, we could split it up into a PositionInterface, OrientationInterface, and SyllableInterface, but not sure what value that adds.  Whichever approach you think is best, I am happy to implement.